### ISSUE TITLE
Explorer Crashing

### ISSUE THREAD

**Issue by ShabbirSemari** [valinet/ExplorerPatcher#1921]
Hi Guys, i do not know why but suddenly my explorer patcher is crashing multiple times. I've updated to the latest version, rolled back to 2 versions below the latest, yet to no avail.. i am living now with the default Win 11 explorer - taskbar and am devastated since i cannot see the open windows in my taskpane!

Can anyone help?

**Comment by ShabbirSemari** [comment]
By mistake closed it.. can any one let me know the solution for this??

**Comment by stanull** [comment]
I have the same issue :(

**Comment by Poopooracoocoo** [comment]
Please state your version of EP and OS build. Closing as duplicate of #1798. There are workarounds that can force EP to download symbols.

**Comment by stanull** [comment]
I was able to make it work, but only after uninstalling EP, removing the whole ExplorerPatcher registry key and installing again. After that the symbols were loaded, and I now have my Win10 taskbar working. But there's still a problem with task view, or win-tab shortcut - it doesn't work.

**Comment by stanull** [comment]
Or after trying this, now alt-tab is not working any more :(

**Comment by stanull** [comment]
EP 22621.1992.56.3, OS 22621.2134

**Comment by Poopooracoocoo** [comment]
> But there's still a problem with task view, or win-tab shortcut - it doesn't work.

That's tracked in #1687. As a workaround, you can use the Windows 11 taskbar instead but it'll introduce another issue. Right clicking the taskbar will crash Windows Explorer (#1805). You can work around it by going to ExplorerPatcher's properties, the advanced page and disabling  "Do not change the taskbar context menu". You can access ExplorerPatcher's properties from your Start Menu or Windows Search.

**Comment by stanull** [comment]
Thx :)
